test/testsupport/stubs/Ttp.scala

```diff
@@ -22,7 +22,7 @@ import testsupport.testdata.TtpJsonResponses
 
 object Ttp {
 
-  private val eligibilityUrl: String = "/time-to-pay/self-serve/eligibility"
+  private val eligibilityUrl: String = "/debts/time-to-pay/eligibility"
```

Comment:
the vals below should be updated with the `/debts` part at the beginning as well shouldn't they?

Reply:
They will do, but I thought I'd do it bit by bit (in the next tickets)

Reply:
you've just updated our own stub to have the `/debts` part in the url for all the api endpoints - the journey will be broken against our own stub if we don't update all of them here I think

Reply:
it would save the hassle of raising multiple PR's if you updated all of the url's here and in the stub at once perhaps

Reply:
Can do I don't mind